projects/npm-tools/packages/npm-scripts/src/scripts/webpack.js

```diff
@@ -68,7 +85,7 @@ function withWebpackConfig(filename, callback) {
 		module.exports = tweakWebpackConfig(
 			webpackConfig,
 			{
-				federation: ${FEDERATION_ENABLED}
+				federation: ${federation}
```

Comment:
I don't know whether I would bother with the pretty printing (ie. the newlines) because when it gets interpolated here the indent is going to be off anyway...

Reply:
I added them just to be safe in case we add something in the future that needs line breaks (or if remove the spaces and turn it into a sum of strings). [OCD](https://en.wikipedia.org/wiki/Obsessive%E2%80%93compulsive_disorder).

Reply:
I've changed it into JSON.